### PR TITLE
mention possibility to bundle SQLite

### PR DESCRIPTION
**Pull request by schlamar**
See https://github.com/diesel-rs/diesel/discussions/3255

### CODE REVIEW

File: diesel/src/lib.rs [hunk 1]
```diff
@@ -101,8 +101,13 @@
...
 
 //!
-//! - `sqlite`: This feature enables the diesel sqlite backend. Enabling this feature requires a compatible copy
-//! of `libsqlite3` for your target architecture.
+//! - `sqlite`: This feature enables the diesel sqlite backend. Enabling this feature requires per default
+//! a compatible copy of `libsqlite3` for your target architecture. Alternatively, you can add `libsqlite3-sys`
+//! with the `bundled` feature as a dependecy to your crate so SQLite will be bundled:
+//! ```
+//! [dependencies]
+//! rusqlite = { version = "0.28.0", features = ["bundled"] }
```
Comment (weiznich) [inline diesel/src/lib.rs:109]:
```suggestion
//! libsqlite3-sys = { version = "0.28.0", features = ["bundled"] }
```

**Review by weiznich — COMMENTED** [review]
Thanks for opening this PR :+1:

I would rather like to suggest to depend on `libsqlite3-sys` instead of depending on `rustqlite` as the later will pull in a otherwise unneeded dependency.

PS: No need to open a new PR for changes to the PR. Just push them to your branch and the PR will update automatically.

**Comment by schlamar** [comment]
> I would rather like to suggest to depend on libsqlite3-sys

Of course. Updated the PR.

**Review by weiznich — APPROVED** [review]

**Comment by weiznich** [comment]
CI failures are unrelated to this change and caused by the clap 4.1 release: https://github.com/diesel-rs/diesel/pull/3479

**Pull request merged**